src/TodoList.js

```diff
@@ -16,7 +17,7 @@ function TodoList() {
   return (
     <ul>
       {todoList.map(function (toDoItem) {
-        return <li key={toDoItem.id}>{toDoItem.title}</li>;
+        return <TodoListItem key={toDoItem.id} title={toDoItem.title} />;
       })}
```

Comment:
we may want to pass toDoItem object as props instead of only passing title 
also we can use arrow function in the map function like below 

```
function TodoList() {
  return (
    <ul>
      {todoList.map((toDoItem)=> <TodoListItem key={toDoItem.id} toDoItem={toDoItem} />)}
    </ul>
  );
}
```

and also update TodoListItem component, and we can destructor props 

```
function TodoListItem({toDoItem}) {
  return (
    <li>{toDoItem.title}</li>
  );
}

```